script/jmeter/01_public_health_dashboard_jmx_test.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'ruby-jmeter'
```

Comment:
Thoughts for keeping these `scripts` separate vs. as rake tasks?

Reply:
The`scripts` folder gets generated when your run `rails g benchmark ...` and I kept that for the both the benchmarking folder and jmeter folder. I wouldn't really mind if we made them rake tasks, but when we want to generate new benchmarks we would just need to manually move them I think.

Reply:
Ah I see. I think it's fine as is. Thank you for answering my question!